.github/workflows/ci.yaml

```diff
@@ -38,13 +38,25 @@ jobs:
       - name: Save artifact
         uses: actions/upload-artifact@v2
         with:
-          name: artifact-${{ github.run_id }}
+          name: artifact_pages-${{ github.run_id }}
           retention-days: 1
           # need to add any root level file to prevent common dirname part trimming
           path: |
             dist
             README.md
 
+      - name: Save target (artifact)
+        uses: actions/upload-artifact@v2
+        with:
+          name: target
+          retention-days: 1
+          # If a wildcard pattern is used, the path hierarchy will be preserved after the first wildcard pattern.
+          # https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions
+          path: |
+            !packages/*/node_modules
+            !packages/*/src
```

Comment:
1) This `path` preset is for monorepo only
2) We have no `target/bundle` in this project, `src` goes to npm publish as is. No bundle — nothing to store.